src/WebJobs.Script/Description/DocumentDBBindingMetadata.cs

```diff
@@ -25,9 +22,9 @@ public override void ApplyToConfig(JobHostConfigurationBuilder configBuilder)
             }
 
             DocumentDBConfiguration config = new DocumentDBConfiguration();
-            if (!string.IsNullOrEmpty(ConnectionString))
+            if (!string.IsNullOrEmpty(Connection))
             {
-                config.ConnectionString = ConnectionString;
+                config.ConnectionString = Utility.GetAppSettingOrEnvironmentValue(Connection);
             }
 
```

Comment:
Call in it JobHostConfigurationBuilder.Done()

---
In reply to: [56897990](https://github.com/Azure/azure-webjobs-sdk-script/pull/133#discussion_r56897990) [](ancestors = 56897990)